<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ADIS_to_ExcelGWBase</class>
 <widget class="QMainWindow" name="ADIS_to_ExcelGWBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ADIS -&gt; GWBase</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="importFileButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ADIS Datei auswählen:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputFile">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>411</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="exportFileButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Excel Ausgabedatei</string>
    </property>
   </widget>
   <widget class="QPushButton" name="doImportButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Konvertierung starten</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mestButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>40</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Messstellen Zuweisung</string>
    </property>
   </widget>
   <widget class="QPushButton" name="paramButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>90</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Parameter Zuweisung</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="outputFile">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>411</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ADIS &lt;-&gt; GWBase</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openExcel">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>170</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Excel Datei öffnen</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="logTextEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>801</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Log:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>862</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
